### DISCUSSION TITLE
Live timing arrow of wind direction not correct

### DISCUSSION THREAD

**Discussion by IvoElferink** [f1multiviewer/issue-tracker#368] [Feature requests] [answered]
See this issue as well: https://github.com/f1multiviewer/issue-tracker/issues/271

The arrow of the wind direction in the live timing (see for example this [screenshot](https://multiviewer.app/_next/static/media/feature-live-timing.45bc2bae.png)) is showing it 180 degrees turned. Wind direction S means from South to Nord, now the arrow in the linked screenshot is indicating that wind direction S means from Nord to South which is not correct.

Can you have this fixed, as once seen you can't unsee it :)

**Comment by f1multiviewer** [comment, marked as answer]
This will be fixed in the next release.
